src/CoreBundle/private/modules/CMSTemplateEngine/views/parts/sharedJS.inc.php

```diff
@@ -24,6 +24,9 @@
         ?>
     $(document).ready(function () {
         <?php
+        /**
+         * @deprecated since 6.3.0 - this message handling is a relict and the template engine has no fields, so this code does nothing.
```

Comment:
Sounds like we can simply remove this snippet?

Reply:
removed the orphaned code